bundles/org.openhab.binding.omnilink/src/main/java/org/openhab/binding/omnilink/internal/handler/OmnilinkBridgeHandler.java

```diff
@@ -612,6 +617,7 @@ private void updateBridgeProperties() {
     @Override
     public void initialize() {
         scheduleReconnectJob();
+        updateChannels();
```

Comment:
Is this a good idea? What if the connection is not present? Wouldn't it make more sense to call `updateChannels` immediately after a successful connection?

Reply:
Yes I thought about that and have changed it in a uncommited update that I am working on.